requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Add Biryani extra requirement here because it is needed by OpenFisca-Core and installation from Git URL below
+# does not take extras_require of Core setup.py into account.
+Biryani[datetimeconv] >= 0.10.4
+--editable git+https://github.com/openfisca/openfisca-core.git#egg=OpenFisca-Core
```

Comment:
I thinked about features which involve changes in all repos (like I did about introspection data). So I needed to switch branch from travis, so I needed to install from git.